File: iniciante/javascript/1094.js
var input = require('fs').readFileSync('stdin', 'utf8')
var lines = input.split('\n')

let countEntradas = 0
let countCobaias = 0
let countCoelhos = 0
let countRatos = 0
let countSapos = 0
let percentCoelhos
let percentRatos
let percentSapos

let qtdEntrada = parseInt(lines.shift())

while(countEntradas++ < qtdEntrada){
    let entradas = lines.shift().split(' ')
    if(entradas[1].replace('\r', '') === 'C')
        countCoelhos += parseInt(entradas[0])
    if(entradas[1].replace('\r', '') === 'R')
        countRatos += parseInt(entradas[0])
    if(entradas[1].replace('\r', '') === 'S')
        countSapos += parseInt(entradas[0])
}

countCobaias = countCoelhos + countRatos + countSapos
percentCoelhos = (countCoelhos * 100)/countCobaias
percentRatos = (countRatos * 100) / countCobaias
percentSapos = (countSapos * 100) / countCobaias


console.log(`Total: ${countCobaias} cobaias`)
console.log(`Total de coelhos: ${countCoelhos}`)
console.log(`Total de ratos: ${countRatos}`)
console.log(`Total de sapos: ${countSapos}`)
console.log(`Percentual de coelhos: ${percentCoelhos.toFixed(2)} %`)
console.log(`Percentual de ratos: ${percentRatos.toFixed(2)} %`)
console.log(`Percentual de sapos: ${percentSapos.toFixed(2)} %`)

File: iniciante/python/1094.py
# -*- coding: utf-8 -*-

count_entrada = 0
total_cobaias = 0
coelho = 0
rato = 0
sapo = 0

qtd_entrada = int(input())
while count_entrada < qtd_entrada:
    entrada = input().split(' ')
    total_cobaias += int(entrada[0])
    if entrada[1] == 'R':
        rato += int(entrada[0])
    elif entrada[1] == 'C':
        coelho += int(entrada[0])
    elif entrada[1] == 'S':
        sapo += int(entrada[0])
    count_entrada += 1

print(f'Total: {total_cobaias} cobaias')
print(f'Total de coelhos: {coelho}')
print(f'Total de ratos: {rato}')
print(f'Total de sapos: {sapo}')
print(f'Percentual de coelhos: {(100 * coelho/total_cobaias):.2f} %')
print(f'Percentual de ratos: {(100 * rato/total_cobaias):.2f} %')
print(f'Percentual de sapos: {(100 * sapo/total_cobaias):.2f} %')
